_posts/2025-30-06-probabl-skolar.md

```diff
@@ -0,0 +1,47 @@
+---
+#### Blog Post Template ####
+
+#### Post Information ####
+title: "Skolar: an open-source initiative to democratize open data science"
+date: June 30, 2025
+
+#### Post Category and Tags ####
+# Format in titlecase without dashes (Ex. "Open Source" instead of "open-source")
+categories:
+  - Updates
+tags:
+  - Open Source
+  - Machine Learning
+
+#### Featured Image ####
+featured-image: BSD_watermark.svg
+
+#### Author Info ####
+# Can accomodate multiple authors
+# Add SQUARE Author Image to /assets/images/author_images/ folder
+postauthors:
+  - name: Pénélope Gittos
+    website: https://www.linkedin.com/in/gittospenelope-data-analyst-growth-bilingual/
+---
+<div>
+  <img src="/assets/images/posts_images/{{ page.featured-image }}" alt="">
+  {% include postauthor.html %}
+</div>
+https://skolar.probabl.ai/
+
+The scikit-learn project always puts efforts on education to build and nurture a strong vibrant open-source community. The goal is straightforward: give everyone, everywhere, the tools they need to easily grasp, engage with, and meaningfully contribute to data science using open-source software. This mission is shared and actively supported by Probabl, a company that helps maintain scikit-learn by employing many of its core contributors and investing in its long-term sustainability. With their support and a deep commitment from the community, we continue building bridges between research, software, and education.
```

Comment:
Maybe we can link to Probabl since it's first appearance:

```suggestion
The scikit-learn project always puts efforts on education to build and nurture a
strong vibrant open-source community. The goal is straightforward: give
everyone, everywhere, the tools they need to easily grasp, engage with, and
meaningfully contribute to data science using open-source software. This mission
is shared and actively supported by [Probabl](https://probabl.ai/), a company
that helps maintain scikit-learn by employing many of its core contributors and
investing in its long-term sustainability. With their support and a deep
commitment from the community, we continue building bridges between research,
software, and education.
```